src/bump-package-version/index.ts

```diff
@@ -5,6 +5,10 @@ export const bumpPackageVersion = async (params: {
   fail: (message: string) => void
   includePaths: Array<string>
 }) => {
+  if(!/^(master|main)$/.test(params.danger.github.pr.base.ref)) {
```

Comment:
@quesabe can we add the banche**s** names in the optional param to the rule function? If the param is empty don't validate the branch.